jni/lib/src/lang/jboolean.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2023, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// Copyright (c) 2023, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// ignore_for_file: overridden_fields
```

Comment:
Is the lint too strict, e.g. with final fields we're only overriding the getter, which should be perfectly fine? https://dart-lang.github.io/linter/lints/overridden_fields.html seems to only talk about non-final fields.

https://github.com/dart-lang/linter/issues/1109 says it's bad practise, because you could have side effects.

Maybe just ignore the lint for the line instead of the whole file, to document that we're intentionally overriding final fields with final fields and have no side effects.